packages/app/src/components/Sidebar/Bookmarks.tsx

```diff
@@ -141,6 +184,7 @@ const Bookmarks = () : JSX.Element => {
                 bookmarkedPage={currentUserBookmark}
                 onUnbookmarked={mutateCurrentUserBookmarks}
                 onRenamed={mutateCurrentUserBookmarks}
+                onDeleted={onBookmarkItemDeleted}
```

Comment:
The method name is not appropriate and please fix the method name like I said before. 
if you have questions, let me know.